### PR TITLE
[IMP] documentation: legal page design

### PR DESCRIPTION
**Pull request by morm-odoo**
Prior to this commit, the terms and conditions were displayed in a table with all the languages and it was not practical for the futur, with many more languages.

task-3073198

### CODE REVIEW

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Looking good! Just a few things to change. :point_down:

File: extensions/odoo_theme/layout_templates/legal.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                    <div class="card-body d-flex flex-column justify-content-between">
+                        <div class="d-flex align-items-center mb-3">
+                            <i class="i-article"></i>
+                            <h5 class="card-title mb-0">{{ _("Odoo Enterprise Agreement") }}
+                                <a class="footnote-reference brackets" href="#id2" id="id1">1</a>
+                            </h5>
+                        </div>
+                        <div>
+                            {%- set enterprise_agreement_path = 'legal/terms/i18n/enterprise_%s.html' % language_code %}
+                                <a class="btn btn-primary" target="_blank" href="{{ enterprise_agreement_path }}" role="button">{{ _("Read") }}</a>
```
Comment (AntoineVDV) [inline extensions/odoo_theme/layout_templates/legal.html:40]:
```suggestion
                            <a class="btn btn-primary" target="_blank" href="{{ enterprise_agreement_path }}" role="button">{{ _("Read") }}</a>
```

File: extensions/odoo_theme/layout_templates/legal.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                            <i class="i-article"></i>
+                            <h5 class="card-title mb-0">{{ _("Odoo Partnership Agreement") }}</h5>
+                        </div>
+                        <div>
+                            {%- set partnership_agreement_path = 'legal/terms/i18n/partnership_%s.html' % language_code %}
+                                <a class="btn btn-primary" target="_blank" href="{{ partnership_agreement_path }}" role="button">{{ _("Read") }}</a>
+                            {%- if language_code != 'en' %}
+                            <a class="btn btn-light text-primary" target="_blank" href="legal/terms/partnership.html">
+                                English
+                            </a>
```
Comment (AntoineVDV) [inline extensions/odoo_theme/layout_templates/legal.html:63]:
```suggestion
                                <a class="btn btn-light text-primary" target="_blank" href="legal/terms/partnership.html">
                                    English
                                </a>
```

File: extensions/odoo_theme/layout_templates/legal.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                            <i class="i-article"></i>
+                            <h5 class="card-title mb-0">{{ _("Terms Of Sale") }}</h5>
+                        </div>
+                        <div>
+                            {%- set terms_of_sale_agreement_path = 'legal/terms/i18n/terms_of_sale_%s.html' % language_code %}
+                                <a class="btn btn-primary" target="_blank" href="{{ terms_of_sale_agreement_path }}" role="button">{{ _("Read") }}</a>
+                            {%- if language_code != 'en' %}
+                            <a class="btn btn-light text-primary" target="_blank" href="legal/terms/terms_of_sale.html">
+                                English
+                            </a>
```
Comment (AntoineVDV) [inline extensions/odoo_theme/layout_templates/legal.html:82]:
```suggestion
                                <a class="btn btn-light text-primary" target="_blank" href="legal/terms/terms_of_sale.html">
                                    English
                                </a>
```

File: extensions/odoo_theme/layout_templates/legal.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                <div class="card h-100">
+                    <div class="card-body d-flex flex-column justify-content-between">
+                        <div class="d-flex align-items-center mb-3">
+                            <i class="i-article"></i>
+                            <h5 class="card-title mb-0">{{ _("Odoo Enterprise Agreement") }}
+                                <a class="footnote-reference brackets" href="#id2" id="id1">1</a>
+                            </h5>
+                        </div>
+                        <div>
+                            {%- set enterprise_agreement_path = 'legal/terms/i18n/enterprise_%s.html' % language_code %}
```
Comment (AntoineVDV) [inline extensions/odoo_theme/layout_templates/legal.html:39]: The path for documents in English is `legal/terms/enterprise.html`. Clicking on Read in English leads to a 404.

File: extensions/odoo_theme/layout_templates/legal.html [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+                            {%- endif %}
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <dl class="footnote brackets">
+            <dt class="label" id="id2">
+                <span class="brackets">
+                    <a class="fn-backref" href="#id1">1</a>
```
Comment (AntoineVDV) [inline extensions/odoo_theme/layout_templates/legal.html:92]: Clicking on this breaks the [1] in the card:

image

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

File: extensions/odoo_theme/layout_templates/legal.html [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+                            <i class="i-article"></i>
+                            <h5 class="card-title mb-0">{{ _("Odoo Enterprise Agreement") }}
+                                <a class="footnote-reference brackets" href="#id2" id="id1">1</a>
+                            </h5>
+                        </div>
+                        <div>
+                            {%- set enterprise_agreement_path = 'legal/terms/i18n/enterprise_%s.html' % language_code %}
+                                <a class="btn btn-primary" target="_blank" href="{{ enterprise_agreement_path }}" role="button">{{ _("Read") }}</a>
+                            {%- if language_code != 'en' %}
+                                <a class="btn btn-light text-primary" target="_blank" href="legal/terms/enterprise.html">
```
[the remark this answers is not in the document]
Comment (morm-odoo) [reply extensions/odoo_theme/layout_templates/legal.html:42]: It's on target blank in the current design, should I change it?

[the remark this answers is not in the document]
Comment (AntoineVDV) [reply extensions/odoo_theme/layout_templates/legal.html:42]: I didn't check the design but, in the doc, internal links are usually not target blank. So I guess yes

**Review by AntoineVDV — APPROVED** [review]

**Review by AntoineVDV — COMMENTED** [review]
@robodoo r+

**Pull request closed without merging**